### PR TITLE
Add VIM built-in package management to the read me

### PR DESCRIPTION
**Pull request by pesarkhobeee**
Dear @scrooloose 
I just updated the README file in order to cover the new feature of VIM version 8 which is built-in package management. hope it brings more ease for new users of your awsome plugin :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ Comment functions so powerful—no comment necessary.
 
 ## Installation
 
+Below are just some of the methods for installing nerdcommenter. Do not follow all of these instructions; just pick your favorite one. Other plugin managers exist, and nerdcommenter should install just fine with any of them.
+
+#### Vim 8+ packages
+
+If you are using VIM version 8 or higher you can use its built-in package management; see `:help packages` for more information. Just run these commands in your terminal:
```
Comment (alerque) [inline README.md]: As far as I understand it this is technically inaccurate. VIM 8 doesn't mange packages, it just gives you a place where you can manage them yourself. See for example [minpac](https://github.com/k-takata/minpac) which specifically wraps this feature with actual package management.

I think I would reword this section including the title so the distinction between manually cloning (and having to manually update) vs. package managers that fetch and update themselves. The issue here is less 3rd party vs. 1st party as it is the kind of function is manual vs. automatic.

**Review by alerque — CHANGES_REQUESTED** [review]
Thanks for the contribution @pesarkhobeee! The original author of this hasn't been active for many years now, I've been managing it for 3 years or so and it was stagnant for a long while before that (see #400 for current status), but yes the author did setup a great plugin.

I agree some instructions for installing manually into plain VIM 8's package path are in order, but I think the actual wording here kind of muddies the water of what a plugin manager actually does. Can you review my notes on that and update accordingly?

File: README.md (hunk 1 above)
Comment (pesarkhobeee) [reply README.md]: Dear @alerque 
Thanks for your detailed description and sorry for my late replay, I was on a trip
I got your point and I totally agree, I will change it
